test/topology_custom/test_mv_topology_change.py

```diff
@@ -77,3 +80,63 @@ async def do_writes(start_it, repeat) -> int:
     stop_event.set()
     await asyncio.gather(*tasks)
 
+# Reproduces #19152
+# Verify a pending replica is not doing unnecessary work of building and sending view updates.
+# 1) we have a table with a materialized view
+# 2) start migrating a tablet from node 1 to node 2
+# 3) while node 2 is a pending replica, write to the table
+# 4) complete migration
+# 5) verify node 2 did not build view updates
+@pytest.mark.asyncio
+@skip_mode('release', 'error injections are not supported in release mode')
```

Comment:
probably should be skipped if running in non tablets mode?

Reply:
IIUC this is not a concern in test.py. Each test suite (or the test case itself, if it creates the cluster manually) decides whether the cluster is using tablets or not.